plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/collection-chooser.tsx

```diff
@@ -21,13 +26,19 @@ import { getCollectionByName } from '../collections';
 import { getDefaultProductCollection } from '../constants';
 
 type CollectionButtonProps = {
-	active?: boolean;
 	title: string;
-	icon: string;
-	description: string;
+	icon: BlockIcon | undefined;
```

Comment:
Let's allow `string` as well, it's technically a correct icon (e.g. `games`, `star-empty`, etc.)

```suggestion
	icon: BlockIcon | string | undefined;
```

Reply:
This is actually already accounted for :smile:

```typescript
type BlockIcon = ReactElement<any, string | React.JSXElementConstructor<any>> | BlockIconNormalized | Dashicon.Icon | ComponentType
```

This is even cooler because it means that string values are type-checked against valid Dashicon options!